test/libsolidity/syntaxTests/using/two_functions_bound_to_the_same_unary_operator_as_global_and_non_global.sol

```diff
@@ -0,0 +1,19 @@
+type Int is int;
+
+using {unsub as -} for Int global;
+using {another_unsub as -} for Int;
+
+function unsub(Int, Int) pure returns (Int) {
+    return Int.wrap(0);
+}
+
+function another_unsub(Int, Int) pure returns (Int) {
+    return Int.wrap(0);
+}
```

Comment:
These won't work as unary operators. They have two arguments. The error should say there are multiple operators but it says that there's no matching operator instead.

```suggestion
function unsub(Int) pure returns (Int) {}
function anotherUnsub(Int) pure returns (Int) {}
```

Also, the `return Int.wrap(0);` bit is superfluous in syntax tests. Same in the other test.

Reply:
I like that too, but in this case it's also about keeping tests concise. I find that a bit distracting in most of these tests and it's not the point of the test anyway.